src/main/java/duke/save/SaveTaskListToText.java

```diff
@@ -0,0 +1,346 @@
+            fileTaskCopy(tasks);
+        }
+
+        return numberOfTasksAdded;
+    }
+
+    // Solution below adapted from
+    // https://stackoverflow.com/questions/5600422/method-to-find-string-inside-of-the-text-file-then-getting-the-following-lines
+    public static boolean checkLineExist(String stringToCheck) {
+        File file = new File("./data/duke.txt");
+
+        try {
+            Scanner scanner = new Scanner(file);
+
+            int lineNum = 0;
+
+            while (scanner.hasNextLine()) {
+                String line = scanner.nextLine();
+                lineNum++;
+
+                if (line.equals(stringToCheck)) {
+                    scanner.close(); // Don't forget!
+                    return true;
+                }
+            }
+            scanner.close();
+        } catch(FileNotFoundException e) {
+            System.out.println("File not found!");
+        }
+
+        return false;
+    }
+
+    // Solution below adapted from https://stackoverflow.com/questions/1625234/how-to-append-text-to-an-existing-file-in-java
+    public static boolean saveToDo(String taskName) {
+        String textToWrite1 = "t-/-0-/-" + taskName;
+        String textToWrite2 = "t-/-1-/-" + taskName;
+
+        try {
+            if (checkLineExist(textToWrite1) == false && checkLineExist(textToWrite2) == false) {
+                String dukeDirectory = "data/duke.txt";
+
+                File file = new File(dukeDirectory);
+                boolean fileIsEmpty = false;
+
+                if (file.length() == 0) {
+                    fileIsEmpty = true;
+                }
+
+                FileWriter addToDo = new FileWriter(dukeDirectory, true); //the true will append the new data
+                if (fileIsEmpty == false) {
+                    addToDo.write("\n");
+                }
+                addToDo.write(textToWrite1);
+                addToDo.close();
+            } else {
+                return false;
+            }
+        } catch(IOException ioe) {
+            System.err.println("IOException: " + ioe.getMessage());
+        }
+
+        return true;
+    }
+
+    // Solution below adapted from https://stackoverflow.com/questions/1625234/how-to-append-text-to-an-existing-file-in-java
+    public static boolean saveEvent(String taskName, String at) {
+        String eventToWrite1 = "e-/-0-/-" + taskName + "-/-/at" + at.substring(3);
+        String eventToWrite2 = "e-/-1-/-" + taskName + "-/-/at" + at.substring(3);
+
+        try {
+            if (checkLineExist(eventToWrite1) == false && checkLineExist(eventToWrite2) == false) {
+                String dukeDirectory = "data/duke.txt";
+                File file = new File(dukeDirectory);
+                boolean fileIsEmpty = false;
+
+                if (file.length() == 0) {
+                    fileIsEmpty = true;
+                }
+
+                FileWriter addToDo = new FileWriter(dukeDirectory, true); //the true will append the new data
```

Comment:
This line is [>120 chars long](https://se-education.org/guides/conventions/java/basic.html#layout) including the comment. It would be good to position the comment above the current line instead.